src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<TabBarIcon /> render 1`] = `<Text />`;
```

Comment:
this looks pretty strange to me, it is only a Text component? Is the icon set via css or what?

Reply:
https://github.com/expo/expo/issues/3566

Reply:
the test file could go right into the component folder, it doesn't needs to be in a `__test__` folder

Reply:
fixed